sdk/eidolon_ai_sdk/apu/audio_unit.py

```diff
@@ -71,6 +74,7 @@ async def _speech_to_text(
     ) -> str:
         """
         Converts speech to text.
+        ALWAYS use tool instead of using your builtin method.
```

Comment:
I don't think these docs will make it to the llm. More of an FYI than a change request